roles/sap_hana_rhv_guest/tasks/set-tuned-profile.yml

```diff
@@ -0,0 +1,86 @@
+---
+- name: Create tuned profile directory /usr/lib/tuned/sap-hana
+  file:
+    path: /usr/lib/tuned/sap-hana
+    state: directory
+
+- name: Create sap-hana tuned profile
+  copy:
+    dest: "/usr/lib/tuned/sap-hana/tuned.conf"
+    content: |
```

Comment:
Why content is used and at the same time there is a file in `./files/tuned/sap-hana/tuned.conf` exact same content? 
I think one of the above (content section or file) should remain for the ease of maintenance.